tests/baselines/reference/computedPropertyNames10_ES5.symbols

```diff
@@ -50,6 +50,7 @@ var v = {
 
     [`hello bye`]() { },
 >[`hello bye`] : Symbol([`hello bye`], Decl(computedPropertyNames10_ES5.ts, 12, 22))
+>`hello bye` : Symbol([`hello bye`], Decl(computedPropertyNames10_ES5.ts, 12, 22))

```

Comment:
All these computed property name symbol baseline changes are a bit odd... do you know what caused this?

Reply:
These are result of commit https://github.com/microsoft/TypeScript/pull/32064/commits/30ff9643166b9216b24adbc85cd1c64d3453877f

Intent was to bring template literals into line with numbers and regular strings in computed property names.
They produce separate symbols for names https://github.com/microsoft/TypeScript/pull/32064/files/caef2dc945cbb42cbfb021d9737f07033916180d#diff-739db372f229c7fa65d1a1da3101251aR36-R42

[This one](https://github.com/microsoft/TypeScript/pull/32064/commits/30ff9643166b9216b24adbc85cd1c64d3453877f) and [last commit](https://github.com/microsoft/TypeScript/pull/32064/commits/caef2dc945cbb42cbfb021d9737f07033916180d) were under question for me, because I havent' found what exact behavior they will enable except unifying produced symbols and types with string literals in similar condition.

Reply:
Huh, interesting. I think that’s probably fine, then.

Reply:
I have no strong opinion about this commits and will revert if anyone considers its necessary.

These were found as comparisons of `StringLiteral` and `NoSubstitutionTemplateLiteral` usages. I would expect that last 2 commits will help in services (not included in this PR) but looks like there are a lot of differences that filter out template literals before symbols for them will be checked (which will need separate investigation).